src/Joonasw.AspNetCore.SecurityHeaders/XContentTypeOptions/XContentTypeOptionsOptions.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Joonasw.AspNetCore.SecurityHeaders.XContentTypeOptions
+{
+    public class XContentTypeOptionsOptions
+    {
+        public XContentTypeOptionsOptions()
+        {
+            
```

Comment:
This constructor should have a comment mentioning that content type sniffing will be disallowed.